src/tools/rustdoc-gui/tester.js

```diff
@@ -100,27 +108,48 @@ async function main(argv) {
 
     let failed = false;
     let files;
+    let tests = [];
     if (opts["files"].length === 0) {
         files = fs.readdirSync(opts["tests_folder"]).filter(file => path.extname(file) == ".goml");
     } else {
         files = opts["files"].filter(file => path.extname(file) == ".goml");
     }
-
+    if (files.length === 0) {
+        console.log("rustdoc-gui: No test selected");
+        process.exit(2);
+    }
     files.sort();
+
+    console.log(`running ${files.length} rustdoc-gui tests`);
+    process.setMaxListeners(files.length + 1);
     for (var i = 0; i < files.length; ++i) {
         const testPath = path.join(opts["tests_folder"], files[i]);
-        await runTest(testPath, options).then(out => {
+        tests.push(runTest(testPath, options));
+    }
+
+    let error_outputs = "";
+    let failed_outputs = "";
+    for (var i = 0; i < tests.length; ++i) {
+        await tests[i].then(out => {
             const [output, nb_failures] = out;
-            console.log(output);
```

Comment:
Instead of removing this, why simply storing all output into an array and then sort it once done before displaying it? Like that it'll remain sorted and everything.

Reply:
see main comment